backend/src/libs/packages/open-ai/open-ai.package.ts

```diff
@@ -0,0 +1,159 @@
+import { Configuration, OpenAIApi } from 'openai';
+
+import { type IConfig } from '~/libs/packages/config/config.js';
+import { type ValueOf } from '~/libs/types/types.js';
+import {
+  type Footer,
+  type Header,
+  type Main,
+  SectionType,
+} from '~/packages/sections/sections.js';
+import { type SiteCreateRequestDto } from '~/packages/sites/sites.js';
+
+import { PROMPT_HEADING, PROMPT_REQUEST } from './libs/constants/constants.js';
+import { PromptExample, PromptRequest } from './libs/enums/enums.js';
+
+class OpenAI {
```

Comment:
this class should know only about api calls but not about the details. everything that is related to the sections, should be in `site` service i guess

Reply:
Same question as above

Reply:
answered